### PR TITLE
HAI-1289 Make it possible in hanke form that hanke start date can be in the past

### PR DESCRIPTION
**Pull request by markohaarni**
# Description

Allow user to input hanke start dates that are in the past by removing validation min rule from start date and allowing form data to be sent even if start date is in the past.

## Type of change

- [ ] Bug fix
- [ ] New feature
- [x] Other

# Checklist:

- [x] I have written new tests (if applicable)
- [x] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
      or other location:

### CODE REVIEW

**Review by alahaitu — APPROVED** [review]

File: src/domain/hanke/edit/HankeForm.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import Form2 from './HankeForm2';
 import FormNotifications from './components/FormNotifications';
 import './HankeForm.styles.scss';
 import { HANKE_SAVETYPE } from '../../types/hanke';
-import { filterEmptyContacts, isHankeEditingDisabled } from './utils';
```
Comment (corvidian) [inline src/domain/hanke/edit/HankeForm.tsx:23]: Can the method `isHankeEditingDisabled` be removed, or is it used elsewhere?

File: src/domain/hanke/edit/HankeForm0.tsx [hunk 2]
```diff
@@ -30,7 +29,6 @@ const Form0: React.FC<FormProps> = ({ errors, register, formData }) => {
 
   return (
     <div className="form0">
-      <EditDisabledNotification formData={formData} />
```
Comment (corvidian) [inline src/domain/hanke/edit/HankeForm0.tsx:33]: There doesn't seem to be other uses for the `EditDisabledNotification` component, can it be removed as well?

File: src/domain/hanke/edit/HankeForm.tsx (hunk 1 above)
Comment (markohaarni) [reply src/domain/hanke/edit/HankeForm.tsx:23]: It's not used elsewhere so removed it.

File: src/domain/hanke/edit/HankeForm0.tsx (hunk 2 above)
Comment (markohaarni) [reply src/domain/hanke/edit/HankeForm0.tsx:33]: EditDisabledNotification component also removed as it is not used anymore.

**Review by corvidian — APPROVED** [review]

**Pull request merged**